pkg/agent/metrics/prometheus.go

```diff
@@ -108,8 +71,12 @@ func InitializePrometheusMetrics(
 	// and will not measure anything unless the collector is first registered.
 	gaugeHost.Set(1)
 
-	ovsStats := newOVSStatManager(ovsBridge, ofClient)
-	if err := legacyregistry.RawRegister(ovsStats); err != nil {
-		klog.Error("Failed to register antrea_agent_ovs_flow_table with Prometheus")
+	// Register	OvsTotalFlowCount metric
```

Comment:
```suggestion
	// Register OVSTotalFlowCount metric
```
But I feel these comments are not really needed as `legacyregistry.Register(OVSTotalFlowCount)` are self-explanatory.